test/unit/format_parse_validators_test.cpp

```diff
@@ -347,6 +347,19 @@ TEST(validator_test, output_file)
         EXPECT_EQ(validator2.get_help_page_message(), "The output file must not exist already and write permissions "
                                                       "must be granted.");
     }
+
+    { // parameter pack constructor - extensions only
```

Comment:
I think a test case is missing: `extensions only as vector`?

Reply:
I think everything is covered?